### ISSUE TITLE
Is it possible to use this package without integrating with SnapChat?

### ISSUE THREAD

**Issue by armanm** [Jobeso/react-native-story-share#19]
I just want to use this with Instagram only but it seems this package requires to have SnapChat SDK installed the . Is this correct?

**Comment by Jobeso** [comment]
Yes, this library requires Snap SDK to be installed alongside. However, there are already some forks removing snapchat, only integrating the Instagram story part.

**Comment by taschik** [comment]
Mind sharing a link to it? It would be great to have this actually configurable to reduce final app size.

**Comment by armanm** [comment]
Thanks @Jobeso. I ended up forking your package to discard the SnapChat dependencies. 

In the end I realised in order for this package to work in my project I have to also downgrade the Gradle version from `5.4.1` to `4.10.2` or else I'd have to deal with this error:

```
* What went wrong:
Could not compile build file android/build.gradle'.
> startup failed:
  General error during semantic analysis: Unsupported class file major version 57

  java.lang.IllegalArgumentException: Unsupported class file major version 57
```

My understanding of Gradle and the Android ecosystem is subzero 😂 Could you tell me what is the implications of downgrading Gradle version in order to use your package?

**Comment by bobber205** [comment]
@armanm You can give my new version of this package a try if you want. https://yarnpkg.com/en/package/react-native-legit-story-share

**Comment by savaskalkan** [comment]
Hey @armanm, I removed snapchat from project you can pull from my fork. 

https://github.com/savaskalkan/react-native-story-share

react native 0.60> (**0.63.3**)
Btw it was not work on ios because of snapchat so I removed it and then worked.
